theme/lumo/vaadin-grid-styles.html

```diff
@@ -29,6 +29,10 @@
         --_lumo-grid-selected-row-color: var(--lumo-primary-color-10pct);
       }
 
+      :host([disabled]) {
+        opacity: 0.7;
```

Comment:
The opacity doesn't show up when testing on iOS 13. Probably not super important though, as interaction with the grid is correctly disabled.

Reply:
I tested with Safari through Browserstack on an iPhone 8 with iOS 13. It doesn't say which version in particular.

Reply:
I guess I know where the problem comes from. I disabled the `animation` property for the case when the grid is disabled. Could you check to see if this fixes the issue?

Reply:
Yep, it seems to work now 👍